src/coreclr/jit/codegenarm64.cpp

```diff
@@ -10444,6 +10444,20 @@ void CodeGen::genArm64EmitterUnitTests()
     theEmitter->emitIns_R_R_R(INS_sve_uxtw, EA_SCALABLE, REG_V17, REG_P6, REG_V14,
                               INS_OPTS_SCALABLE_D); /* UXTW    <Zd>.D, <Pg>/M, <Zn>.D */
 
+    // IF_SVE_GA_2A
+    theEmitter->emitIns_R_R_I(INS_sve_sqrshrn, EA_SCALABLE, REG_V0, REG_V0, 5,
```

Comment:
nit: since all of these encode in the 2x form, try passing different register combination like `REG_V30`, `REG_16`, etc.